internal/ingress/controller/store/store.go

```diff
@@ -479,6 +479,18 @@ func New(checkOCSP bool,
 					if key == configmap {
 						store.setConfig(cm)
 					}
+
+					ings := store.listers.IngressAnnotation.List()
+					for _, ingKey := range ings {
+						key := k8s.MetaNamespaceKey(ingKey)
+						ing, err := store.GetIngress(key)
+						if err != nil {
+							glog.Errorf("could not find Ingress %v in local store: %v", key, err)
+							continue
+						}
+						store.extractAnnotations(ing)
+					}
```

Comment:
@aledbf did you see https://github.com/kubernetes/ingress-nginx/pull/2656#discussion_r197060345?

Reply:
@ElvinEfendi yes, please check my comment